cmd/kube-controller-manager/app/options/options_test.go

```diff
@@ -295,6 +297,7 @@ func TestAddFlags(t *testing.T) {
 				ConcurrentGCSyncs: 30,
 				GCIgnoredResources: []garbagecollectorconfig.GroupResource{
 					{Group: "", Resource: "events"},
+					{Group: eventv1.GroupName, Resource: "events"},
```

Comment:
Does the "" on the line above not act as a wildcard?

Reply:
hm maybe it doesn't. That is embarrassing to be wrong.

Reply:
No, it doesn't.  it's the empty group.

Reply:
I don't even want to know how long it's been like this.